app/src/ui/window/full-screen-info.tsx

```diff
@@ -85,11 +85,7 @@ export class FullScreenInfo extends React.Component<
     this.setState({ renderTransitionGroup: false })
   }
 
-  private renderFullScreenNotification() {
-    if (!this.state.renderInfo) {
-      return null
-    }
-
+  private renderFullScreenNotification(): JSX.Element {
```

Comment:
The last hunk you used `JSX.Element | null` as the function return type, should we use that here?

Reply:
That was suitable for the last function because it was either returning `CSSTransitionGroup` (if it has a commit to render) or `null`. This case is slightly different, and I'd probably inline it into `render()` if it's at all confusing because it's a very trivial function...